content/docs/getting-started.md

```diff
@@ -18,109 +18,109 @@ redirect_from:
+हम अनुरोध करते है की आप [इस जावास्क्रिप्ट के ओवरव्यू को देखे](https://developer.mozilla.org/en-US/docs/Web/JavaScript/A_re-introduction_to_JavaScript) ताकि आप अपने जानकारी के स्तर को आक सकें | इसमें आपको आधे से एक घंटा लगेगा लेकिन आप React सीखने में अधिक आत्मविश्वास महसूस करेंगे |
 
->Tip
+>सुझाव 
 >
->Whenever you get confused by something in JavaScript, [MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript) and [javascript.info](https://javascript.info/) are great websites to check. There are also [community support forums](/community/support.html) where you can ask for help.
+>जब भी आप किसी जावास्क्रिप्ट की चीज़ की वजह से उलझन में पड जाएं तो [MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript) और  [javascript.info](https://javascript.info/) अपने आप सही करने के लिए बहुत अछि वेबसीटे हैं | [कम्युनिटी सहायता मंच](/community/support.html) पे आप मदद माँग सकते हैं |
 
-### Practical Tutorial {#practical-tutorial}
+### प्रैक्टिकल टुटोरिअल {#practical-tutorial}
 
-If you prefer to **learn by doing,** check out our [practical tutorial](/tutorial/tutorial.html). In this tutorial, we build a tic-tac-toe game in React. You might be tempted to skip it because you're not building games -- but give it a chance. The techniques you'll learn in the tutorial are fundamental to building *any* React apps, and mastering it will give you a much deeper understanding.
+अगर आप **कर के** सीखना पसंद करते हैं तो हमारा [प्रैक्टिकल टुटोरिअल देखें](/tutorial/tutorial.html). इस टुटोरिअल में हम एक टिक-टैक्-टो का गेम बनाते हैं | हो सकता है आपको ऐसा लगे की हमें ये छोड़ देना चाहिए क्योकि हम कोई गेम नहीं बना रहे -- लेकिन इसे एक मौका दें | इसमें सीखी गई विधियां *किसी भी* React ऐप को बनाने में काम आएगी, और इसका अच्छा अभ्यास आपको React की गहरी समझ देगा | 
 
-### Step-by-Step Guide {#step-by-step-guide}
+### स्टेप-बाय-स्टेप गाइड {#step-by-step-guide}
 
-If you prefer to **learn concepts step by step,** our [guide to main concepts](/docs/hello-world.html) is the best place to start. Every next chapter in it builds on the knowledge introduced in the previous chapters so you won't miss anything as you go along.
+अगर आप React के **कांसेप्ट को स्टेप-बाय-स्टेप सीखना** पसंद करते हैं तो हमारी [मुख्य कॉन्सेप्ट्स की गाइड](/docs/hello-world.html) शुरू करने की लिए एक उत्तम जगह है | इसमें हर अगला अध्याय पिछले अध्यायों पर बना है तो इससे आप से कुछ छूटेगा नहीं |
 
-### Thinking in React {#thinking-in-react}
+### React में सोचें {#thinking-in-react}
 
-Many React users credit reading [Thinking in React](/docs/thinking-in-react.html) as the moment React finally "clicked" for them. It's probably the oldest React walkthrough but it's still just as relevant.
+बहुत सारे React के यूज़र्स [React में सोचे](/docs/thinking-in-react.html) पढ़ने को उनकी React कि समझ बढ़ाने में मदद करने का क्रेडिट देते हैं, वह समझते हैं कि इसको पढ़ने के बाद हि उन्हें React समझ में आई | ये React के सबसे पुराने वॉकथ्रू में से एक है | 
```

Comment:
कि -> badi maatra

Reply:
हि -> same

Reply:
but it's still just as relevant got lost in translation?

Reply:
the bottom 2 ones here haven't been fixed here @yashguptaz

Reply:
I have fixed those now, can you mark any missed ups unresolved?

Reply:
समझते हैं कि इसको पढ़ने के बाद one should not have been changed

Reply:
It's probably the oldest React walkthrough but it's still just as relevant. -> but it's still just as relevant has not been translated at all